io.github.openWMail.json

```diff
@@ -0,0 +1,62 @@
+ {
+    "app-id": "io.github.openWMail",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "stable",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "/app/openwmail/openWMail",
+    "separate-locales": false,
+    "rename-desktop-file": "openwmail.desktop",
+    "rename-icon": "openWMail",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=xdg-download",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "modules": [
+        {
+            "name": "openWMail",
+            "buildsystem": "simple",
+            "build-commands": [
+                "ar x openWMail_*.deb",
+                "rm -f openWMail_*.deb",
+                "tar xf data.tar.gz",
+                "rm -f control.tar.gz data.tar.gz debian-binary",
+                "cp -r usr/* opt/* /app",
+                "install -D openWMail.512x512.png /app/share/icons/hicolor/512x512/apps/openWMail.png",
+                "chmod -R a-s,go+rX,go-w /app/openwmail",
+                "chmod +x /app/openwmail/openWMail",
+                "desktop-file-edit --set-key=Exec --set-value='/app/openwmail/openWMail %U' /app/share/applications/openwmail.desktop",
+                "install -Dm644 io.github.openWMail.appdata.xml /app/share/appdata/io.github.openWMail.appdata.xml"
+            ],
+            "sources": [
+                {
+                    "type": "file",
```

Comment:
Add tarball with source code too to make it available via `Sources` extension.

Reply:
done :+1: